packages/lib-classifier/src/plugins/drawingTools/experimental/components/LineControls/LineControls.js

```diff
@@ -0,0 +1,179 @@
+  const point = (activePosition == 'tr')
+    ? { x: x + (50 / scale), y: y + (50 / scale) }
+    : { x: x + w - (50 / scale), y: y + (50 / scale) }
+
+  const StyledPath = styled('path')`
+    fill: ${FILL_COLOR};
+    &:hover {
+      cursor: pointer;
+      fill: ${HOVER_COLOR};
+    }
+  `
+
+  const TL = `M ${point.x - OUTER_RADIUS} ${point.y}
+    A ${OUTER_RADIUS} ${OUTER_RADIUS} 0 0 1 ${point.x} ${point.y - OUTER_RADIUS}
+    L ${point.x} ${point.y - INNER_RADIUS}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 0 ${point.x - INNER_RADIUS} ${point.y}
+    Z`;
+
+  const TR = `M ${point.x + OUTER_RADIUS} ${point.y}
+    A ${OUTER_RADIUS} ${OUTER_RADIUS} 0 0 0 ${point.x} ${point.y - OUTER_RADIUS}
+    L ${point.x} ${point.y - INNER_RADIUS}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 1 ${point.x + INNER_RADIUS} ${point.y}
+    Z`;
+
+  const BL = `M ${point.x - OUTER_RADIUS} ${point.y}
+    A ${OUTER_RADIUS} ${OUTER_RADIUS} 0 0 0 ${point.x} ${point.y + OUTER_RADIUS}
+    L ${point.x} ${point.y + INNER_RADIUS}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 1 ${point.x - INNER_RADIUS} ${point.y}
+    Z`;
+
+  const BR = `M ${point.x + OUTER_RADIUS} ${point.y}
+    A ${OUTER_RADIUS} ${OUTER_RADIUS} 0 0 1 ${point.x} ${point.y + OUTER_RADIUS}
+    L ${point.x} ${point.y + INNER_RADIUS}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 0 ${point.x + INNER_RADIUS} ${point.y}
+    Z`;
+
+  const C = `M ${point.x - INNER_RADIUS} ${point.y}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 0 ${point.x + INNER_RADIUS} ${point.y}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 0 ${point.x - INNER_RADIUS} ${point.y}
+    Z`;
+
+  function move() {
+    setActivePosition((activePosition == 'tl') ? 'tr' : 'tl');
+  }
+
+  function destroy() {
+    onDelete();
+  }
+
+  function onEnterOrSpace(ev, func) {
+    if (ev.keyCode === 13 || ev.keyCode === 32) func()
+  }
+
+  return (
+    <g
+      ref={ref}
+      role='group'
+      aria-label={'Line Controls'}
+      focusable='false'
+	  transform={`rotate(${-rotate} ${width / 2} ${height / 2})`}
+
+    >
+      <g
+        role='button'
+        aria-label='Undo'
+        focusable='true'
+        tabIndex='0'
+      >
+        <Tooltip label='Undo'>
+          <StyledPath
+            d={TL}
+            stroke={ARROW_STROKE_COLOR}
+            strokeWidth={STROKE_WIDTH}
+            onPointerDown={mark.undo}
+            onKeyDown={(ev) => { onEnterOrSpace(ev, mark.undo) }}
+          ></StyledPath>
+        </Tooltip>
+      </g>
+      <g
+        role='button'
+        aria-label='Redo'
+        focusable='true'
+        tabIndex='0'
+      >
+        <Tooltip label='Redo'>
+          <StyledPath
+            d={TR}
+            stroke={ARROW_STROKE_COLOR}
+            strokeWidth={STROKE_WIDTH}
+            onPointerDown={mark.redo}
+            onKeyDown={(ev) => { onEnterOrSpace(ev, mark.redo) }}
+          ></StyledPath>
+        </Tooltip>
+      </g>
+      <g
+        role='button'
+        aria-label='Close'
```

Comment:
Needs localisation. There might already be a localised 'Close' string in the language dictionaries.